xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -4218,11 +4216,11 @@ void CDVDPlayer::GetAudioStreamInfo(int index, SPlayerAudioStreamInfo &info)
   }
 }
 
-int CDVDPlayer::AddSubtitleFile(const std::string& filename, const std::string& subfilename, CDemuxStream::EFlags flags)
+int CDVDPlayer::AddSubtitleFile(const std::string& filename, const std::string& subfilename)
```

Comment:
this method must not be called from the outside because it accesses demuxer. currently AddSubtitle calls this directly. it should send a player message instead